lonestar/analytics/gpu/independentset/mis.cu

```diff
@@ -196,14 +197,12 @@ __global__ void gg_main_pipe_1_gpu(CSRGraphTy gg, int STEPS, unsigned int* prio,
     *cl_STEPS = STEPS;
   }
 }
+*/
 void gg_main_pipe_1_wrapper(CSRGraphTy& gg, int& STEPS, Shared<unsigned int>& prio, PipeContextT<WorklistT>& pipe, dim3& blocks, dim3& threads)
 {
-  if (false)
-  {
+#if 1
```

Comment:
Better to remove unused code instead of commenting it out.

Reply:
removed